js/components/interface/query/queryBuilder.js

```diff
@@ -1047,8 +1047,15 @@ define(function (require) {
               continue;
             }
             if (args.columnsPresent[counter] == "images") {
-              instance[args.columnsPresent[counter]] = JSON.parse(record[args.columnsPresent[counter]]).initialValues[0].value.elements[0].initialValue.data;
-              continue;
+              if (record[args.columnsPresent[counter]].indexOf('{') > -1){
```

Comment:
@Robbie1977 I would change this check inside the if with a function to check if the data can be parsed to be sure the error is handled anyway with a function similar to the one I am putting here below, and use this function to check inside the if.
The rest is good for me.
Thx! 
```
var isJson = data => {
          try {
            return (JSON.parse(data) && data);
          } catch (e) {
            return false;
          }
        }
```